redash/query_runner/prometheus.py

```diff
@@ -22,6 +56,10 @@ def configuration_schema(cls):
     def annotate_query(cls):
         return False
 
+    @classmethod
+    def type(cls):
+        return "Prometheus"
```

Comment:
This is not necessary and a bad idea, because it will break for all the existing users who already created data sources for Prometheus (where type is `prometheus`).

Reply:
@arikfr fixed, please have another review